.github/workflows/python-mypy.yml

```diff
@@ -11,7 +11,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.7, 3.8, 3.9, 3.10, 3.11]
```

Comment:
Try putting `' '` around `3.10` I have noticed it is being interpreted as `3.1` causing the build to fail.

Reply:
Ah right xD